src/pattern-library/components/patterns/data/TablePage.tsx

```diff
@@ -155,71 +156,76 @@ export default function TablePage() {
                 <code>false</code>
               </Library.InfoItem>
             </Library.Info>
-            <Library.Demo title="Table with stickyHeader and Scroll" withSource>
+            <Library.Demo
+              title="Table with stickyHeader and rounded Scroll"
```

Comment:
What was the reason for picking this example to add rounding to - because it was already customizing the scroll container?

Reply:
No particular reason. Just to avoid having so many very similar examples.

I'm open to keep this one as is and add an extra example with a rounded scroll.